Iterators and Closures/Standard Library Types/Intro/task.md

```diff
@@ -262,6 +269,7 @@ We implemented the `Iterator` trait by defining the `next` method, so we can now
 For example, if for some reason we wanted to take the values produced by an instance of `Counter`, pair them with values produced by another `Counter` instance after skipping the first value, multiply each pair together, keep only those results that are divisible by 3, and add all the resulting values together, we could do so, as shown in the test below:
```

Comment:
For example, if for some reason we wanted to take the values produced by an instance of `Counter`, pair them with values produced by another `Counter` instance after skipping the first value, multiply each pair together, keep only those results that are divisible by 3, and add up all the resulting values together, we could do so as shown in the test below: